python-main.js

```diff
@@ -1283,9 +1283,17 @@ function web_editor(config) {
     }
 
     function webusbErrorHandler(err) {
+        // Hide flashing modal
+        $("#flashing-info").addClass('hidden');
```

Comment:
I've removed this line since it's not needed anymore after 9985e325f539451c80bad24e3b039869e9094c44, however nowhere else the `flashing-info` is hidden, so wondering if this is still needed somewhere? We've never had an issue with this div showing the wrong content before.